website/Dockerfile

```diff
@@ -69,4 +69,26 @@ RUN HUGOHOME="$(mktemp -d)" \
     && chmod +x /usr/local/bin/hugo \
     && rm -r "${HUGOHOME}"
 
+
+# Install gcloud
+# Make sure that your operating system meets the requirements
```

Comment:
Why do we need changes to the docker image?

Reply:
To complete the upload to GCS step and to run the script `append_index_html_to_internal_links.py` inside the container we had to make those changes.